NativeRefTests/FXSnapshotTestCase.swift

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+class FXSnapshotTestCase: FBSnapshotTestCase {
```

Comment:
@fluffyemily Created a subclass that contains the check for the environment variable so we don't have to write this in all of our test cases.